### DISCUSSION TITLE
Sending external eve.json into security onion

### DISCUSSION THREAD

**Discussion by cjbrick** [Security-Onion-Solutions/securityonion#2204] [Unsupported Versions] [answered]
Hi folks,

I've got a standalone SO server configured at home in a lab environment prior to deployment at work. We've got a firewall running suricata and would like to push the eve.json data from there into security onion and I'm not seeing a simple answer in the documentation. Would this be accomplished by sending the eve.json to the logstash beat port 5044?

Thanks for any guidance you may have. Security Onion is incredibly well done. I've barely scratched the surface of it and I'm blown away by how great the 2.3.x release is. 

Thanks,

Chris

**Comment by weslambert** [comment, marked as answer]
You should be able to pick this up with Filebeat and send it to the manager without issue.

**Reply by cjbrick** [reply]
Thank you! I'll give it a try.
